api-docs/list-clients.md

```diff
@@ -0,0 +1,34 @@
+# View a list of clients
+
+API is provided to query a list of clients from the database.
+
+##Request
+GET /api/clients/?firstName=Mario&lastName=Luigi&zip=84107&page=1
```

Comment:
You can remove the slash after `clients/` -- API endpoints do not need trailing slashes.